### ISSUE TITLE
Not Sure...

### ISSUE THREAD

**Issue by TehMerkMods** [marzika/Snapprefs#136]
Im not sure if this is an issue 
But the build you are working for is 9.31.1.0
You are most likely fully aware that the latest release is 9.36.0.0
so let's say you are making everything work for 9.31.1.0 isnt that completley pointless due to the fact that 9.36.0.0 is the relased snapchat version? Would also be nice if you could add some sort of auto update (to your hooks/variables/methods) like maybe write a function that could possibly update everything itself with the installed version of snapchat :package: 

Anyways reading this it seems more like a question and a suggestion but hey

The issue for me is why build of 9.31.1.0 when 9.36.0.0 is released :palm_tree:

**Comment by edi194** [comment]
When every feature'll work on 9.31.1.0 then hooks'll be updated to 9.36.0.0. 
We've here only 9.31.1.0 version, marz said he is using 9.35.x.x

**Comment by alexhulbert** [comment]
I don't currently have a phone to test with, but I figure auto-updates should be possible with at least a few features. I built a Java library that auto-updated back a couple years ago. IIRC it was fairly difficult to get perfectly, but it certainly beat always updating my library to support new versions. I might be able to dig it up and see what library I used, but the point is that tools do exist out there that allow you to use pattern-matching-like systems to automatically hook into certain methods. I apologize for the poor wording on that, but you get my point.
